<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_window</class>
 <widget class="QFrame" name="new_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>add rule</string>
  </property>
  <widget class="QLabel" name="HOOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HOOK_POINT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="S_IP">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>S_IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="D_IP">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>D_IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="S_MASK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>S_MASK</string>
   </property>
  </widget>
  <widget class="QLabel" name="D_MASK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>D_MASK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="S_IP_LINE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="D_IP_LINE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="S_MASK_LINE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="D_MASK_LINE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PUSH">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PUSH</string>
   </property>
  </widget>
  <widget class="QComboBox" name="HOOK_POINT">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>input</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>output</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>forward</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>S_PORT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>D_PORT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="S_PORT_LINE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="D_PORT_LINE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CLICK">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>点击查看使用说明</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PROTOCOL</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PROTOCOL">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UDP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ICMP</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
